src/test-e2e/po/meta-card.po.ts

```diff
@@ -21,6 +26,14 @@ export class MetaCard extends Component {
     });
   }
 
+  getMetaCardItems(): promise.Promise<MetaCardItem[]> {
```

Comment:
Just a heads up, we have `MetaDataItemComponent` in `meta-data-item.po.ts` now that at some point we should tie in with this function.